storage_service/locations/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class LocationsAppConfig(AppConfig):
+    default_auto_field = "django.db.models.AutoField"
+    name = "locations"
```

Comment:
Now that this exists I think we can move [this import](https://github.com/artefactual/archivematica-storage-service/blob/6196e34ad20b66544e2426cd9be7551103efae99/storage_service/locations/models/__init__.py#L2-L6) here.

What do you think of creating a `ready` method that imports the signals like the `CommonAppConfig.ready` method does?

Reply:
From https://docs.djangoproject.com/en/3.2/ref/applications/#django.apps.AppConfig.ready

> Subclasses can override this method to perform initialization tasks such as registering signals.

That's exactly what the `import` in that `__init__` file is trying to do (see the comment block above the `import` line). But the `AppConfig` subclass is the appropriate place to do that. See all our other `AppConfig` subclasses in AM and SS. Some of them do exactly that.